plugins/basic/tk_create_menus/generateMenuItems.py

```diff
@@ -27,17 +27,29 @@ def __init__(
         self._functionTemplate ="""        [MenuItem("Shotgun/{menuName}", false, {priority})] 
         public static void MenuItem{count}()
         {{
-            Service.Call("execute_menu_item", "{submenuName}");
+            Call("{submenuName}");
         }}"""
 
         self._fileContentsTemplate = """
 #if SHOTGUN_INSTALLED
 //If Unity version for this project is lower than v2020.3,Removing the Shotgun for Unity package leaves this script in the assets/Shotgun directory.
 // It is recommended to manually delete the assets/Shotgun directory after removing the Shotgun for Unity package from a project.
+using Python.Runtime;
+using UnityEditor.Scripting.Python;
 namespace UnityEditor.Integrations.Shotgun
 {{
     class {className}
     {{
+        public static void Call(string menuItem)
+        {{
+            PythonRunner.EnsureInitialized();
+            using (Py.GIL())
+            {{
+                dynamic sg_bootstrap = PythonEngine.ImportModule("sg_bootstrap");
+                sg_bootstrap.test_execute_menu_item(menuItem);
```

Comment:
here is another "test_" method name.. Could you rename it? No need for me to review again if you change it

Reply:
I love that you refactored that part by the way :-)